charts/ctms/templates/background_deployment.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "ctms.fullname" . }}
+  labels:
+    app.kubernetes.io/component: background
+    {{- include "ctms.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: background
+      {{- include "ctms.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        app.kubernetes.io/component: background
+        {{- include "ctms.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "ctms.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
```

Comment:
Is it worth including a minimal page that returns 200 as a livenessProbe/readinessProbe? If python fails to start entirely, or dies to a memory leak or the like, would be nice to try deleting the container.  (probably ok to defer until later)

Reply:
If python fails to start or crashes later, won't k8s detect this and try to restart the pod (until you hit crash loop backoff)? I'd think livenessprobe is only useful if you worry the app might be running but deadlock, and readinessprobe isn't meaningful for an app that isn't receiving traffic.

I do think it could make sense to have some http endpoints in the job though, such as for external healthchecks or exposing metrics.

Reply:
In taskcluster we use the livenessprobe to [restart all containers once a day](https://github.com/taskcluster/taskcluster/commit/c8bc3d57a02ed47bb23da7256a5866e7cb93fa9d). Now I'm kinda wondering if we want to do that here too

Reply:
I'll do that in a followup if we do it at all.